src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -1454,6 +1455,7 @@ def db_export(
         database_name=database_name,
         server_name=server_name,
         resource_group_name=resource_group_name,
+        no_wait=no_wait,
```

Comment:
This is not how `--no-wait` works.

SDK doesn't accept this parameter, this is a poller framework implemented by CLI core.

Have you tested this new parameter?

Reply:
@evelyn-ys Should I use `sdk_no_wait ` function from `azure.cli.core.util ` instead ?

Reply:
fixed